src/lib/components/collections/collections-list/ImportCollection.svelte

```diff
@@ -283,9 +333,9 @@
   textarea {
     width: 100%;
     resize: vertical;
-    height: 80%;
-    max-height: 100%;
+    height: 100px !important;

```

Comment:
use calc function to set height of the textarea

Reply:
resolved